package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "start": "ts-node server/server.ts",
     "build": "webpack --mode production",
-    "test": "jest"
+    "test": "jest",
+    "it": "cypress open"
```

Comment:
how does it detect which test to run?

Reply:
what you mean? 
script from package.json should open our mainTest?

Reply:
jest is scanning for all *.test.* files and executing them.

It here opens cypress, I think that better would be if it executed `cypress run`.
However we need to put regex here to mention what tests should be executed by cypress run.